pkg/ddevapp/backdrop.go

```diff
@@ -0,0 +1,143 @@
+package ddevapp
+
+import (
+	"fmt"
+	"os"
+	"path/filepath"
+	"text/template"
+
+	"github.com/Masterminds/sprig"
+	"github.com/drud/ddev/pkg/appports"
+	"github.com/drud/ddev/pkg/output"
+	"github.com/drud/ddev/pkg/util"
+)
+
+type BackdropSettings struct {
+	DatabaseName     string
+	DatabaseUsername string
+	DatabasePassword string
+	DatabaseHost     string
+	DatabaseDriver   string
+	DatabasePort     string
+	DatabasePrefix   string
+	HashSalt         string
+	Signature        string
+}
+
+// NewBackdropSettings produces a BackdropSettings object with default values.
+func NewBackdropsettings() *BackdropSettings {
```

Comment:
Same here, why not just use the identical Drupal version?

Reply:
:+1:

Reply:
I do think we can consider renaming so it doesn't imply drupal.